saito-rust/src/main.rs

```diff
@@ -781,18 +781,20 @@ pub async fn run_utxo_to_issuance_converter(threshold: Currency) {
     let slip_type = "Normal";
 
     for (key, value) in &data {
-        if value >= &threshold {
-            let key_base58 = bs58::encode(key).into_string();
-            file.write_all(format!("{}\t{}\t{}\n", value, key_base58, slip_type).as_bytes())
-                .await
-                .expect("failed writing to issuance file");
-        }
+        let key_base58 = if threshold < 25000 {
+            PROJECT_PUBLIC_KEY.to_string()
+        } else {
+            bs58::encode(key).into_string()
+        };
+        file.write_all(format!("{}\t{}\t{}\n", value, key_base58, slip_type).as_bytes())
```

Comment:
this will add multiple entries for project public key I think. those values should be aggregated and later written for a single entry with project key. @davikstone2